ecs-cli/modules/config/readwriter_v1.go

```diff
@@ -169,7 +169,7 @@ func readProfileConfig(path string, profileConfigKey string, localConfig *LocalC
 // This function either reads the old single configuration file
 // Or if the new files are present, it reads from them instead
 func (rdwr *YAMLReadWriter) Get(clusterConfig string, profileConfig string) (*LocalConfig, error) {
-	localConfig := &LocalConfig{}
+	localConfig := &LocalConfig{Version: yamlConfigVersion}
```

Comment:
@PettitWesley @SoManyHs I wanted to bring this statement to attention as I had to make a change to get the integ test to run. I believe this was a bug because this method belongs to a YAMLReadWriter and instead it would create a LocalConfig where the Version defaulted to `iniConfigVersion` which is legacy

Reply:
Hm. That is true, though I am confused about how this allows the tests to still pass  since it looks like the rest of this method involves trying to read the LocalConfig as an INI file and mutating LocalConfig:
```
	// try to readINI first; it is either successful or it
	// set localConfig to be its default value (all fields empty strings)
	readINI(rdwr.destination, localConfig)

	// Try to read the config as YAML
	// nothing will happen if it fails
	errYAML := readClusterConfig(configPath, clusterConfig, localConfig)
```

Perhaps @PettitWesley can shed more insight?